libbeat/reader/readjson/docker_json.go

```diff
@@ -144,13 +144,13 @@ func (p *DockerJSONReader) parseDockerJSONLog(message *reader.Message, msg *logL
 	if err != nil {
 		return errors.Wrap(err, "parsing docker timestamp")
 	}
+	message.Ts = ts
```

Comment:
This line doesn't have to be moved I guess 😄

Reply:
Although this line has no direct impact on fix but this line is moved to maintain the logical order of the code (here in case of timestamp related operation) because I think maintaining logical order in source code is also important for readability.